client/src/ingredients/browser-compatibility-table/rows.js

```diff
@@ -1,13 +1,14 @@
 import React from "react";
 import { BrowserSupportDetail } from "./browser-support-detail";
 import { BrowserSupportNotes } from "./browser-support-notes";
+import { Link } from "@reach/router";
 
 function buildCompatibilityObject(query, compatibilityData) {
   const features = {};
 
   if (!!compatibilityData.__compat) {
     const name = query.split(".").pop();
-    features[name] = compatibilityData.__compat;
+    features[name] = { ...compatibilityData.__compat, ...{ origin: true } };
```

Comment:
I don't get it. Why would it ever not be `origin` and what does that even mean?

Reply:
Yeah I'll get a comment in for this, basically the first row in the BCD table represents the compatibility support for the page you're currently on. I checked the prod site and it doesn't make those links. So i added another field to the first object so that when it renders it can differentiate that *this* is the origin so *don't* make it a link.

This block specifically pulls out the current pages compatibility and sets the name (there is no name given like the others in the data it's just __compat so i got it from the query).